lib/seahorn/BvOpSem2TrackingRawMemMgr.hh

```diff
@@ -20,14 +21,10 @@ namespace details {
 // Currently this implementation has a metadata memory word size of 1 byte.
 // For every byte written to conventional memory, we set the corresponding
 // metadata memory address to value 1.
-class TrackingRawMemManager : public OpSemMemManagerBase {
+class TrackingRawMemManager : public CoreMemManager {
 private:
-  RawMemManager m_main;
-  RawMemManager m_metadata;
-
-  static const unsigned int g_MetadataBitWidth = 8;
-  static const unsigned int g_MetadataByteWidth = g_MetadataBitWidth / 8;
-  static const unsigned int g_num_slots = 2;
+  OpSemMemManagerMixin<RawMemManager> m_main;
```

Comment:
readability can be improved if throughout the code the `Mixin` are hidden under some defined type. So that lines like these are read as 'RawMemManager m_main` or similar.

Reply:
done